### DISCUSSION TITLE
Changes and translations disappeared after updating the App

### DISCUSSION THREAD

**Discussion by Eta-Beta** [moudey/Shell#424] [Q&A] [answered]
Good morning, as per the title, today I updated Nilesoft from version 1.8 to the latest via Winget. The problem is that all the changes made and translations from the English language have disappeared, so I proceeded to replace the imports folder and the shell.nss file with the modified ones but the program generates an error. The problem is certainly the new version of Nilesoft which certainly has some differences from the ver. 4,4.. changes that I can't resolve. If anyone could help me I would be grateful. Thanks anyway for a reply.

**Comment by RubicBG** [comment, marked as answer]
https://github.com/moudey/Shell/issues/422#issuecomment-1951706681
look in shell.log to see which line contains the error(s). static{ } and dynamic are no longer used
